hypershift-operator/controllers/hostedcluster/internal/platform/powervs/powervs.go

```diff
@@ -26,6 +26,23 @@ const (
 type PowerVS struct {
 }
 
+func (p PowerVS) ReconcileExternalCCM(ctx context.Context, c client.Client, hcluster *hyperv1.HostedCluster, createOrUpdate upsert.CreateOrUpdateFN, kubeConfig *corev1.Secret, ccmImage string) (err error) {
```

Comment:
Please add some unit tests for this. The code you added doesn't work at all due to the incorrect usage of `createOrUpdate`, something like that needs to be caught in a unittest